easybuild/easyconfigs/n/nanodbc/nanodbc-2.14.0-GCCcore-11.2.0.eb

```
@@ -0,0 +1,30 @@
+# ll4il Lorentz Institute
+easyblock = 'CMakeMake'
+
+name = 'nanodbc'
+version = '2.14.0'
+
+homepage = 'https://github.com/nanodbc/nanodbc'
+description = "A small C++ wrapper for the native C ODBC API."
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = ['https://github.com/%(name)s/%(name)s/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['56228372042b689beccd96b0ac3476643ea85b3f57b3f23fb11ca4314e68b9a5']
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('CMake', '3.22.1'),
+    ('Boost', '1.77.0', '', ('GCC', '11.2.0')),
+]
+
+separate_build_dir = True
+configopts = " -DBUILD_SHARED_LIBS=TRUE -DNANODBC_ENABLE_BOOST=ON "
+
+sanity_check_paths = {
+    'files': ['lib/libnanodbc.so'],
```

we should avoid hard-coding the shared library extension by using the `SHLIB_EXT` template constant
```suggestion
    'files': ['lib/libnanodbc.%s' % SHLIB_EXT],
```